src/toil/wdl/wdltoil.py

```diff
@@ -1452,7 +1489,14 @@ def get_file_paths_in_bindings(environment: WDLBindings) -> List[str]:
     """
 
     paths = []
-    map_over_files_in_bindings(environment, lambda x: paths.append(x))
+
+    def append_to_paths(path: str) -> Optional[str]:
+        # Append element and return the element. This is to avoid a logger warning inside map_over_typed_files_in_value()
```

Comment:
Maybe we should just stop issuing that warning if we're having to return values we don't care about just to avoid it?